src/simulations/local_elec.py

```diff
@@ -556,8 +559,15 @@ def policy_to_rewardvar(
             bool_penalty, date, loads, E_req_only, h, last_step, home_vars)
 
         if self.prm['grd']['manage_voltage']:
+            start = time.time()
             hourly_line_losses, voltage = self.network.pf_simulation(home_vars['netp'])
+            end = time.time()
             voltage_squared = np.square(voltage)
+            if self.prm['grd']['computational_burden_analysis']:
+                time_to_solve_pp = end - start
+                self.record.timing_hourly_pandapower(time_step=h, timer_pandapower=time_to_solve_pp)
```

Comment:
I mean you can create a separate method, but also if we created a brand new method for each piece of data we want to save in the `record` object that would be a lot of methods. My approach has been to group the quantities I want to save at the end of each `epoch` (`record.end_epoch`), for the final `epoch` of each `repeat` only (`record.last_epoch`), at the end of each `repeat` (`record.save(end_of='repeat')`) or at the very end (`record.save(end_of='end')`). Here because we are probably only interested one average time for all the times each of these steps have been performed in the whole of an experiment, my suggestions above have been to just add the average/std in `record.save(end_of='repeat')`.